topologies/basic.json

```diff
@@ -1,17 +1,16 @@
 {
-    "enableExplorer": false,
-    "enableHub": false,
-    "nodes": 2,
-    "notifiers": 0,
-    "tokens": [
-        {
-            "symbol": "LUM",
-            "amount": 1
-        },
-        {
-            "symbol": "RIF",
-            "amount": 2
-        }
-    ]
+  "enableExplorer": false,
```

Comment:
formatting changes here only

Reply:
Be careful, you change from 4 spaces to 2, 2 is not being used anymore, 4 is better for the eye.

Reply:
this was automatically done by the extension. can we maybe find a source to determine what the convention is here?

Reply:
[RLP-1089](https://jirainfuy.atlassian.net/browse/RLP-1089)